cmd/test.go

```diff
@@ -47,15 +49,21 @@ func test(ctx context.Context) error {
 		return err
 	}
 
+	v := version.Current()
+	logger.WithFields(logrus.Fields{
+		"GitCommit":  v.GitCommit,
+		"AppVersion": v.AppVersion,
+	}).Info("running task: test")
+
 	// Just used to verify fleet-scheduler can authenticate
 	_, err = client.New(otelCtxWithCancel, cfg, logger)
 	if err != nil {
 		return err
 	}
 
 	// purge secrets from config before printing the config (for debug purposes)
-	cfg.FdbCfg.ClientSecret = ""
-	cfg.CoCfg.ClientSecret = ""
+	cfg.FdbCfg.ClientSecret = "REDACTED"
+	cfg.CoCfg.ClientSecret = "REDACTED"
```

Comment:
if you change this here and only did a shallow copy of the config object in `client.New` we might have just clobbered things. Be careful when you test this in prod.